RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -322,8 +324,6 @@ struct ManageSubscriptionButton: View {
                                  product: promotionalOfferData.product,
                                  promoOfferDetails: promotionalOfferData.promoOfferDetails)
         })
-        .buttonStyle(ManageSubscriptionsButtonStyle())
-        .disabled(self.viewModel.loadingPath != nil)
```

Comment:
Hmm I'm not sure I understand why doing this before actually fixes this... But I'm ok with the fix 👍

Reply:
It's because it applies it to the sheet as well, and the sheet is the one presenting the promo offer, so basically everything in the sheet is disabled

Reply:
Ahh makes sense. Thanks for explaining!